### PR TITLE
Followup edits to PR 1950

### PR DESCRIPTION
**Pull request by bfallonf**
#1950

Rev history in original PR

@ewolinetz How does the reader set the ES_HOST (and others) variables? A quick google says it's a file, but if it's an environment variable like you say in the original PR is that not done with a command? Other than that, no questions. Thanks!

### CODE REVIEW

**Comment by ewolinetz** [comment]
@bfallonf all of those are set by editing the Elasticsearch dc with `oc edit dc/{DC}`

**Comment by bfallonf** [comment]
@ewolinetz Cool. Thanks. I've added that into the section.

@ahardin-rh @adellape @tnguyen-rh @tpoitras Available for peer review please.

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -741,37 +741,45 @@ $ oc get route  --all-namespaces --selector logging-infra=support
...
+logs to `OPS_HOST`.
 
-Fluentd knows where to send its logs to based on the `ES_HOST`, `ES_PORT`,
-`OPS_HOST` and `OPS_PORT` environment variables.  If you have an external
-Elasticsearch instance that will contain both application and operations logs,
-ensure that `ES_HOST` and `OPS_HOST` are the same and that `ES_PORT` and
-`OPS_PORT` are also the same.  Fluentd is configured to send its application logs
-to the `ES_HOST` destination and all of its operations logs to `OPS_HOST`.
+To direct logs to a specific Elasticsearch instance, edit the deployment
+configuration and replace the value of the above variables to the desired
```
Comment (tnguyen-rh) [inline install_config/aggregate_logging.adoc]: Hmm, "replace  ... to the desired instance" is new usage to me.  How about "replace ... with the desired instance"?

**Comment by tnguyen-rh** [comment]
@bfallonf One nit, otherwise LGTM.

File: install_config/aggregate_logging.adoc (hunk 1 above)
Comment (bfallonf) [inline install_config/aggregate_logging.adoc]: @tnguyen-rh Yup. Good eye.

**Comment by bfallonf** [comment]
@tnguyen-rh Cool. Thanks. I'll merge.

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -741,37 +741,45 @@ $ oc get route  --all-namespaces --selector logging-infra=support
...
 endif::openshift-origin[]
 
 ifdef::openshift-enterprise[]
-You can use `oc edit dc/logging-fluentd` to update your Fluentd configuration.
-It is advised that you first scale down your number of replicas to 0 before
-editing the DeploymentConfig.
+You can use `oc edit dc/logging-fluentd` to update your Fluentd configuration,
+making sure to first scale down your number of replicas to zero before editing
+the deployment configuration.
 enddef::openshift-enterprise[]
```
Comment (adellape) [inline install_config/aggregate_logging.adoc]: Just noticed this, currently in prod.

File: install_config/aggregate_logging.adoc [hunk 3]
```diff
@@ -741,37 +741,45 @@ $ oc get route  --all-namespaces --selector logging-infra=support
 ----
 ====
 
-== External Elasticsearch instance with Fluentd
+[[sending-logs-to-an-external-elasticsearch-instance]]
+== Sending Logs to an External Elasticsearch instance
```
Comment (adellape) [inline install_config/aggregate_logging.adoc]: s/instance/Instance/

**Comment by adellape** [comment]
Couple comments inline that I came across while publishing.

**Comment by tnguyen-rh** [comment]
@adellape Thanks for catching these.

https://github.com/openshift/openshift-docs/pull/2555 is for OSE 3.2.  This is the normal origin first, then cherry-pick on release flow.

https://github.com/openshift/openshift-docs/commit/a6b48b0d5391908dfb23a0c123f7dd2c08708a40 is for OSE 3.1.  This is a direct push to branch `enterprise-3.1`.  I did it this way because a cherry-pick would probably introduce merge conflicts.

WDYT?  Am i missing anything?

**Comment by adellape** [comment]
@tnguyen-rh Seems perfecto to me! I don't see the `enddef` in master, enterprise-3.2, or enterprise-3.1. Thanks.

**Pull request merged**